src/search.js

```diff
@@ -3,3 +3,37 @@ import React from "react";
 import axios from "axios";
 
 import "./css/song.css";
+export default class SongSearch extends React.Component{
+	constructor(){
+		super()
```

Comment:
Let's work on this PR together in class to fix the format (indentation, spaces, etc) before we merge this to the `main` branch